### PR TITLE
RTL notification area mark as read icon

### PR DESCRIPTION
**Pull request by Scout4all**

### CODE REVIEW

File: phpBB/styles/prosilver/theme/bidi.css [hunk 1]
```diff
@@ -332,7 +332,12 @@ li.breadcrumbs span:first-child > a {
 .rtl a.feed-icon-forum {
 	float: left;
 }
-
+/* Notification mark read link */
+.rtl .dropdown-extended a.mark_read {
+    border-radius: 0 3px 3px 0;
+    right:auto;
```
Comment (nickvergessen) [inline phpBB/styles/prosilver/theme/bidi.css]: `right: auto;`

File: phpBB/styles/prosilver/theme/bidi.css [hunk 2]
```diff
@@ -332,7 +332,12 @@ li.breadcrumbs span:first-child > a {
...
+      left: 0;
```
Comment (nickvergessen) [inline phpBB/styles/prosilver/theme/bidi.css]: please use tabs for intendation

**Comment by nickvergessen** [comment]
Would be cool, if you can fix the commit message:
https://wiki.phpbb.com/Git#Commit_Messages

**Comment by prototech** [comment]
Replaced by https://github.com/phpbb/phpbb/pull/3428

**Pull request closed without merging**